app/views/categories/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<% if @category.errors.any? %>
+  <ul class="errors">
+    <% @category.errors.each do |column, message| %>
+      <li>
+        <strong><%= column.capitalize %></strong> <%= message %>
+      </li>
+    <% end %>
+  </ul>
+<% end %>
+<ul>
+<p>Current Categories:</p>
+<% @categories.each do |category| %>
+<li><%= category.name %></li>
+<% end %>
+</ul>
+
+<%= form_for @category do |f| %>
+<!-- use the instance to give form_for all the information they need to know about tasks! -->
+  <%= field_set_tag 'Add New Category', :html => {:class => "form-group"}  do %>
+    <%= f.text_field :name, placeholder: "Name" %>
+    <%= f.submit %>
+
```

Comment:
BUG: I was able to create lots of categories with no name by clicking on submit without putting anything in the text field.

How could you fix this in the view? In the model?